docs/contacts/email-identifier.md

```diff
@@ -0,0 +1,3 @@
+# Email as Unique Identifier
+
+Contact’s unique identifier is email address, meaning, you can have communication coming from various sources like both an email and a form and they will all associate with each other because of the matching email address. This also allows all users to see contact communication across all team members, as when those communications are logged, they will associate with the contact record.
```

Comment:
`Contact’s unique identifier is email address, meaning, you can have communication coming from various sources like both an email and a form and they will`
Edit: A contact's unique identifier is their email address. You can have communication coming from various sources, such as an email and a form, and they will....